src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -107,26 +109,29 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
     }
 
     public render() {
-        
+
         return (
             <If condition={this.isChartVisible}>
-                <div className={styles.chart}
+                <div className={styles.studyViewChart}
                      onMouseEnter={this.handlers.onMouseEnterPlot}
                      onMouseLeave={this.handlers.onMouseLeavePlot}>
-                    <ChartHeader 
+                    <ChartHeader
                         clinicalAttribute={this.props.clinicalAttribute}
                         showControls={this.mouseInPlot}
-                        showResetIcon={this.props.filters.length>0}
+                        showResetIcon={this.props.filters.length > 0}
                         handleResetClick={this.handlers.resetFilters}
                     />
-
-                    <If condition={this.props.chartType===ChartType.PIE_CHART}>
-                        <PieChart
-                            ref={this.handlers.ref}
-                            onUserSelection={this.handlers.onUserSelection}
-                            filters={this.props.filters}
-                            data={this.data} />
-                    </If>
+                    <div style={{position: 'relative', height: 'calc(100% - 20px)'}}>
```

Comment:
Not sure about this height calculation.  Why not have a fixed height and then padding?